csharp/ql/test/library-tests/csharp9/globalStmt.expected

```diff
@@ -0,0 +1,11 @@
+statements
+| GlobalStmt.cs:7:1:25:1 | {...} |
+| GlobalStmt.cs:11:1:11:23 | ...; |
+| GlobalStmt.cs:12:1:12:23 | ...; |
+| GlobalStmt.cs:13:1:13:4 | ...; |
+| GlobalStmt.cs:15:1:17:1 | M(...) |
+globalBlock
+| GlobalStmt.cs:7:1:25:1 | {...} | GlobalStmt.cs:7:1:25:1 | <Main>$ | file://:0:0:0:0 | args | GlobalStmt.cs:7:1:25:1 | <Program>$ |
```

Comment:
The locations are slightly unexpected here. `<Main>$` is overlapping the `Attr` declaration. Also, `args` has no location, it could be assigned to the correct file at least.

Reply:
Pushed an extra commit to fix the location of `args`.